sdk/core/System.ClientModel/tests/Pipeline/HttpClientPipelineTransportTests.cs

```diff
@@ -372,7 +372,7 @@ public async Task UnbufferedResponseContentStreamEmptyIfNoResponseContent()
 
         await transport.ProcessSyncOrAsync(message, IsAsync);
 
-        Assert.AreEqual(string.Empty, BinaryData.FromStream(message.Response!.ContentStream!).ToString());
```

Comment:
https://github.com/dotnet/runtime/issues/68262

This was fixed in 8.0.0, we should upgrade to 8.0.0 once 6.0.0 is out of support. Then this test can be changed back. Will open a tracking issue for things to change when we upgrade to 8.0.0